x-pack/test/alerting_api_integration/security_and_spaces/group4/tests/alerting/get_rule_execution_kpi.ts

```diff
@@ -17,7 +17,8 @@ export default function getRuleExecutionKpiTests({ getService }: FtrProviderCont
 
   const retry = getService('retry');
 
-  describe('getRuleExecutionKpi', () => {
+  // failing on CI, this file wasn't loaded in any config
```

Comment:
@XavierM  looked at some blames, seems like this one was specifically not run for a while, but the "global" version got enabled at some point.  Maybe add a comment to the file if we know we're not using it at the moment?  Or maybe it was mistakenly not enabled?